langwatch/prisma/migrations/20250403125535_add_elasticsearch_hybrid/migration.sql

```diff
@@ -0,0 +1,16 @@
+-- AlterTable
+ALTER TABLE "Monitor" RENAME CONSTRAINT "Check_pkey" TO "Monitor_pkey";
+
+-- AlterTable
+ALTER TABLE "Organization" ADD COLUMN     "elasticsearchApiKey" TEXT,
+ADD COLUMN     "elasticsearchNodeUrl" TEXT,
+ADD COLUMN     "useElasticsearch" BOOLEAN NOT NULL DEFAULT false;
```

Comment:
maybe we could rename it to `useCustomElasticsearch`? a bit annoying to change I know, but since this didn't go to production yet, you can change manually in the migration here, in the schema, and rename the dev db column directly